Multi-Chain/kava-evm-cosmos-multi-chain/schema.graphql

```diff
@@ -0,0 +1,29 @@
+# To improve query performance, we strongly suggest adding indexes to any field that you plan to filter or sort by
+# Add the `@index` or `@index(unique: true)` annotation after any non-key field
+# https://academy.subquery.network/build/graphql.html#indexing-by-non-primary-key-field
+
+type Transfers @entity {
+  id: ID!
+  blockHeight: BigInt
+  txHash: String
+  fromAddress: String
```

Comment:
Is it possible to map Cosmos Addresses to their Eth equivalent, if so, can you make an `Account` entity with this mapping?